src/components/Header.jsx

```diff
@@ -100,10 +100,10 @@ export function Header() {
                         >
                           <div className="space-y-4">
                             <Services/>
-                            <MobileNavLink href="#pricing">
+                            <MobileNavLink href="pricing">
                               Pricing
                             </MobileNavLink>
-                            <MobileNavLink href="#reviews">
+                            <MobileNavLink href="#contact">
```

Comment:
Why are we not linking the HTML pages?

Reply:
let's merge it for now.
this PR is nothing significant in it.